src/components/common/PeopleTable.tsx

```diff
@@ -255,7 +269,7 @@ const PeopleTable: React.FC<PeopleTableProps> = ({
                 setSelectedParticipants([]);
               }
             }}
-            disabled={selectedParticipants.length === 0}
+            disabled={!showSelectedUsers && selectedParticipants.length === 0}
```

Comment:
I wonder if this was not supposed to be || instead of &&

Reply:
I will double check this. Good catch!